unlock-app/package.json

```diff
@@ -54,6 +54,7 @@
     "react-jazzicon": "1.0.4",
     "react-jwt": "1.1.8",
     "react-markdown": "8.0.5",
+    "react-simple-wysiwyg": "2.1.4",
```

Comment:
Please let's just use markdown for now. I worry about the data model here and added complexity.

Reply:
by using markdown, we also need to convert the markdown to HTML before we  add the custom content to the email template. is this ok?  
also, in this case, we need to add 2 new packages instead of one. (1 for markdown to save the template + 1 to transform the markdown in HTML to add the custom content)

Reply:
> by using markdown, we also need to convert the markdown to HTML before we add the custom content to the email template. is this ok?

Absolutely!

> also, in this case, we need to add 2 new packages instead of one. (1 for markdown to save the template + 1 to transform the markdown in HTML to add the custom content)

We actually already have the markdown package as it is used for the events page for example?